src/wasm.h

```diff
@@ -686,6 +686,21 @@ class Literal {
   }
 };
 
+// Literal type subclasses for Web IDL
```

Comment:
I think these serve a webidl-bindings purpose only. So it would be better to not have them in the core `wasm.h`. file. Perhaps in the same directory as the idl file?